package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "organizacionsemanal",
+  "version": "1.0.0",
+  "description": "## Descripción del problema",
+  "main": "index.js",
```

Comment:
Tratad de borrar la parte del boilerplate que no uséis, como esto.

Reply:
Resuleto en https://github.com/carlotiii30/organizacionSemanal/pull/5/commits/1fe10ec30dcbbf6787d02c32a2067d12180159da